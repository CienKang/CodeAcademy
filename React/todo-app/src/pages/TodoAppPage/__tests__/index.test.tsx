import { render } from '@testing-library/react';
import { TodoAppPage } from '..';
import React from 'react';


describe('TodoAppPage', () => {

    it('should render the todo app page', () => {

        const { getByText } = render(<TodoAppPage />);
        expect(getByText('To Do APP')).toBeInTheDocument();
    });

    it('should match the snapshot when rendered for the first time', () => {

        const { asFragment } = render(<TodoAppPage />);
        expect(asFragment()).toMatchSnapshot();
    });

});