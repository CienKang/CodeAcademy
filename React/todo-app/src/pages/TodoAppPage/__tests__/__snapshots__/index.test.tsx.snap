// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoAppPage should match the snapshot when rendered for the first time 1`] = `
<DocumentFragment>
  <div
    class="todo-container"
  >
    <h1>
       To Do APP
    </h1>
    <div
      class="search-bar"
    >
      <input
        placeholder="Enter the task name"
      />
      <button>
        Add
      </button>
    </div>
    <div
      class="todo-container"
    />
  </div>
</DocumentFragment>
`;
